rules/experimental/rbe/exec_properties.bzl

```diff
@@ -13,6 +13,96 @@
 # limitations under the License.
 
 """This file contains macros to create and manipulate dictionaries of properties to be used as execution properties for RBE.
+
+It also contains macros that create repository rules for standard and custom sets of execution
+properties.
+
+Here are some examples of how to use these repository rules:
+
+Scenario 1 - The standard use case:
+
+In the WORKSPACE file, call
+  rbe_exec_properties(
+      name = "exec_properties",
+  )
+
+This creates a local repo @rbe_exec_properties with standard RBE execution property constants. For
+example, NETWORK_ON which is the dict {"dockerNetwork" : "standard"}
+
+Then, in some BUILD file, you can reference this execution property constant as follows:
+
+  load("@exec_properties//:constants.bzl", "NETWORK_ON")
+  ...
+  exec_properties = NETWORK_ON
+
+The reason not to directly set exec_properties = {...} in a target is that then it might be hard to
+depend on such a target from another repo, if, say, that other repo wants to use remote execution
+but not RBE.
+
```

Comment:
Should we have a pointer here to the SoT for which are the "standard RBE execution property constants"

Reply:
done